### DISCUSSION TITLE
security association created but tunnel not installed

### DISCUSSION THREAD

**Discussion by abbbel2** [strongswan/strongswan#395] [Q&A]
//// ipsec.conf
conn one-to-two
  auto=start
  #compress=no
  type=tunnel
  keyexchange=ikev2
  authby=secret
  #rightauth=psk
  #leftauth=psk
  left=xx.xx.xx.xx
  leftid=xx.xx.xx.xx
  leftsubnet=xx.xx.xx.xx
  right=xx.xx.xx.xx
  rightid=xx.xx.xx.xx
  #rightsubnet=xx.xx.xx.xx
  rightsubnet=xx.xx.xx.xx
  ike=aes256-sha256-modp1024!
  esp=aes256-sha256!
  ikelifetime=86400s
  lifetime=28800s
  dpddelay=30s
  dpdtimeout=30s
  dpdaction=restart
  #keyingtries=3
  #forceencaps=yes

/// ipsec statusall
Security Associations (1 up, 0 connecting):
one-to-two[2]: ESTABLISHED 15 minutes ago, xx.xx.xx.xx[xx.xx.xx.xx]...xx.xx.xx.xx[xx.xx.xx.xx]
one-to-two[2]: IKEv2 SPIs: 51a9c07f4490b75b_i* f2fb2225be6e180b_r, pre-shared key reauthentication in 23 hours
one-to-two[2]: IKE proposal: AES_CBC_256/HMAC_SHA2_256_128/PRF_HMAC_SHA2_256/MODP_1024

// ipsec up one-to-two
establishing CHILD_SA one-to-two{39}
generating CREATE_CHILD_SA request 38 [ SA No TSi TSr ]
sending packet: from xx.xx.xx.xx[4500] to xx.xx.xx.xx[4500] (208 bytes)
received packet: from xx.xx.xx.xx[4500] to xx.xx.xx.xx[4500] (80 bytes)
parsed CREATE_CHILD_SA response 38 [ N(NO_PROP) ]
received NO_PROPOSAL_CHOSEN notify, no CHILD_SA built
failed to establish CHILD_SA, keeping IKE_SA
establishing connection 'one-to-two' failed

// router log
input-interface: Internal
input-status: up
input-line-status: up
output-interface: Conn-To-Intr
output-status: up
output-line-status: up
Action: drop
Drop-reason: (acl-drop) Flow is denied by configured rule

**NB** i have another connection with ikev1 that works

**Comment by tobiasbrunner** [comment]
As the log tells you, the peer rejects the CREATE_CHILD_SA request (what happens during IKE_AUTH?). `esp=aes256-sha256!` could be wrong (but it could also be any number of other reasons as NO_PROPOSAL_CHOSEN acts as kind of a default error notify). Check with the peer.

**Reply by abbbel2** [reply]
i have checked with peer and all seems to be fine. Is there any possible issue other than that like firewall configuration?

**Comment by abbbel2** [comment]
Fixed my issue, with this https://wiki.strongswan.org/issues/2590
